lib/OpenQA/Task/Git/Clone.pm

```diff
@@ -106,6 +112,11 @@ sub _git_clone ($job, $ctx, $path, $url) {
         return;
     }
 
+    unless (_git_is_workdir_clean($path)) {
+        $ctx->warn("NOT updating dirty git checkout at $path");
```

Comment:
would be nice to add a prompt. I assume that the checkout is skipped and need cleaning, right?

Reply:
What do you mean with a prompt here?

Reply:
I mean add something like "Need to clean $path." or something

Reply:
To me the error message is clear enoufg